test/flint/fq-test.jl

```diff
@@ -1,3 +1,21 @@
+function test_elem(R::FqFiniteField)
+   n = rand(fmpz(0):order(R))
+   iszero(n) && return zero(R)
+   return gen(R)^n
```

Comment:
```suggestion
   return rand(R)
```
The previous code leads one to believe that `gen(R)` returns a generator for the multiplicative group, which it usually does but doesn't necessarily do (sorry :(). In order to not miss any elements, the `rand` function should work just fine.
We could also increase the probability of returning zero, which might test more corner cases...